controllers/localization_controller.go

```diff
@@ -233,7 +233,8 @@ func (r *LocalizationReconciler) reconcile(ctx context.Context, obj *v1alpha1.Lo
 	}
 
 	// create snapshot
-	snapshotName := fmt.Sprintf("%s/snapshots/%s:%s", r.OCIRegistryAddr, obj.Spec.SnapshotTemplate.Name, obj.Spec.SnapshotTemplate.Tag)
+	snapshotName := fmt.Sprintf("%s/snapshots/%s/%s/%s/%s", r.OCIRegistryAddr, componentVersion.Spec.Component, componentVersion.Spec.Version, configResource.Name, configResource.Version)
```

Comment:
We will be constructing our own url and repository names because we don't have a digest at this point only at the later stage once the layer has already been pulled.